src/db/postgres/lrsql/postgres/record.clj

```diff
@@ -59,7 +60,13 @@
       (migrate-xapi-statement-timestamps! tx))
     (when-not (some? (query-xapi-statement-stored-exists tx))
       (alter-xapi-statement-add-stored! tx)
-      (migrate-xapi-statement-stored-times! tx)))
+      (migrate-xapi-statement-stored-times! tx))
+    (when-not (some? (query-state-document-last-modified-is-timestamptz tx))
```

Comment:
I was going ask why you needed this separate query until I saw that you're using three different functions to do the PG migration. Even then I think you could've just added the conditions directly in the commands, but I guess this way is clearer, especially if you're just looking at the Clojure code.

Reply:
yeah i mean it just skips the whole chunk in the future.